### PR TITLE
added snowpipe scipts

### PR DESCRIPTION
**Pull request by hemanthmereddy**
## Approach

This script creates snowpipe for the ingestions of json files from s3 to snowflake automatically. 

https://apptega-atlanta.atlassian.net/browse/CORE-5020

## Testing

After the code is run, the aws s3 json files should be loaded into snowflake target table.

## [Peer Review Questions](https://apptega-atlanta.atlassian.net/wiki/x/AYAPiQ)

\- [ ] The code does what it is supposed to do per the JIRA

\- [ ] The code is formatted according to [our standards](https://apptega-atlanta.atlassian.net/wiki/x/AQDFgQ)

\- [ ] The method and variable names make sense and they are as self explanatory as possible ($solution instead of $item)

\- [ ] As far as I'm aware this is the most efficient way to write this code. I have checked new queries use the ORM when possible

\- [ ] This code has error handling and logging as appropriate

\- [ ] The critical portions of the functionality are tested per [unit testing standards](https://apptega-atlanta.atlassian.net/wiki/x/H4C8kQ)

\- [ ] This code follows the ["defensive coding"](https://en.wikipedia.org/wiki/Defensive_programming) standards

\- [ ] I have added comments for any questions I have about the code

### CODE REVIEW

File: snowpipe/snowpipe.sql [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+-- Create pipe 
+CREATE OR REPLACE PIPE s3_to_snf_from_azure_pipe
+AUTO_INGEST = FALSE
+AS
+COPY INTO cloudquery.public.raw_azure_storage_containers_data
+FROM @s3_stage_from_azure
+FILE_FORMAT = 'json_load_format';
+
+-- Create the data landing table
+CREATE OR REPLACE TABLE cloudquery.public.azure_storage_containers(
```
Comment (adi-apptega) [inline snowpipe/snowpipe.sql:30]: Will we need all these columns? 

image

Could we use a VARIANT type for `properties`?

Comment (adi-apptega) [reply snowpipe/snowpipe.sql:30]: This is what the JSON objects look like in S3. I think we can simplify this by using the JSON keys as columns

image

Comment (hemanthmereddy) [reply snowpipe/snowpipe.sql:30]: you mean don't we need everything in a separate column?

Comment (adi-apptega) [reply snowpipe/snowpipe.sql:30]: With VARIANT columns we can store JSON objects. For now I think making a VARIANT column for properties will be more manageable in the long run

Comment (hemanthmereddy) [reply snowpipe/snowpipe.sql:30]: is this what we are aiming for?
SELECT 
  $1:"_cq_id"::varchar AS _cq_id,
  $1:"_cq_parent_id"::varchar AS _cq_parent_id,
  $1:"_cq_source_name"::varchar AS _cq_source_name,
  $1:"_cq_sync_time"::varchar AS _cq_sync_time,
  $1:"extended_location"::varchar AS extended_location,
  $1:"id"::varchar AS id,
  $1:"identity"::varchar AS identity,
  $1:"kind"::varchar AS kind,
  $1:"location"::varchar AS location,
  $1:"name"::varchar AS name,
  $1:"properties_accessTier"::variant AS properties_accessTier,
  $1:"sku_name"::varchar AS sku_name,
  $1:"sku_tier"::varchar AS sku_tier,
  $1:"subscription_id"::varchar AS subscription_id,
  $1:"tags"::varchar AS tags,
  $1:"type"::varchar AS type
FROM azure_storage_containers_stream;

Comment (adi-apptega) [reply snowpipe/snowpipe.sql:30]:
```
_cq_id STRING,
_cq_parent_id STRING,
_cq_source_name STRING,
_cq_sync_time TIMESTAMP,
etag STRING,
id STRING,
name STRING,
properties VARIANT,
subscription_id STRING,
type STRING
```

**Review by adi-apptega — APPROVED** [review]

**Pull request merged**